### PR TITLE
Sieve_of_Eratosthenes in PHP Issue #2174

### PR DESCRIPTION
**Pull request by jalajdorai**
Issue #2174  

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Sieve_of_Eratosthenes in PHP

#### Languages Used:
- PHP

#### Files Added:
- Sieve_of_Eratosthenes.php

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by jalajdorai** [comment]
@jainaman224  Can you review this Sieve_of_Eratosthenes in PHP which was assigned to me??

**Comment by jainaman224** [comment]
Add issue number.

File: Sieve_Of_Eratosthenes/Sieve_of_Eratosthenes.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+   
+    $n = 30; 
+    echo "Following are the prime numbers "
+     ."smaller than or equal to " .$n."\n" ; 
+    SieveOfEratosthenes($n); 
+   
+?> 
+
+//Following are the prime numbers below 30
+//2 3 5 7 11 13 17 19 23 29
```
Comment (jainaman224) [inline Sieve_Of_Eratosthenes/Sieve_of_Eratosthenes.php]: Add signle new line at the end of file.

File: Sieve_Of_Eratosthenes/Sieve_of_Eratosthenes.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+} 
+   
+    $n = 30; 
+    echo "Following are the prime numbers "
+     ."smaller than or equal to " .$n."\n" ; 
+    SieveOfEratosthenes($n); 
+   
+?> 
+
+//Following are the prime numbers below 30
```
Comment (jainaman224) [inline Sieve_Of_Eratosthenes/Sieve_of_Eratosthenes.php]: Add space after `//`.

File: Sieve_Of_Eratosthenes/Sieve_of_Eratosthenes.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        } 
+    } 
+  
+    
+    for ($p = 2; $p <= $n; $p++) 
+        if ($prime[$p]) 
+            echo $p." "; 
+} 
+   
+    $n = 30;
```
Comment (jainaman224) [inline Sieve_Of_Eratosthenes/Sieve_of_Eratosthenes.php]: Incorrect indentation.

**Comment by jalajdorai** [comment]
@jainaman224 I have made the changes please review.

**Comment by SakshiShreya** [comment]
Please squash your commits.

**Comment by somya-kapoor** [comment]
@jalajdorai Are you working on it?

**Comment by jalajdorai** [comment]
@somya-kapoor Yes I am working on it . I have already done the necessary changes

**Comment by jalajdorai** [comment]
@SakshiShreya  I have squashed my commits.I think I have done all the necessary changes

**Comment by jalajdorai** [comment]
@jainaman224  @SakshiShreya  @somya-kapoor  Please review this PR.

File: Sieve_Of_Eratosthenes/Sieve_of_Eratosthenes.php [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?php 
+function SieveOfEratosthenes($n) 
+{  
+    $prime = array_fill(0, $n+1, true); 
+    for ($p = 2; $p*$p <= $n; $p++)
```
Comment (singh-shreya6) [inline Sieve_Of_Eratosthenes/Sieve_of_Eratosthenes.php:5]: Please add space around operators. i.e * here.

File: Sieve_Of_Eratosthenes/Sieve_of_Eratosthenes.php [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<?php 
+function SieveOfEratosthenes($n) 
+{  
+    $prime = array_fill(0, $n+1, true); 
+    for ($p = 2; $p*$p <= $n; $p++) 
+    {     
+        if ($prime[$p] == true) 
+        { 
+            for ($i = $p*$p; $i <= $n; $i += $p)
```
Comment (singh-shreya6) [inline Sieve_Of_Eratosthenes/Sieve_of_Eratosthenes.php:9]: Space around *

**Comment by singh-shreya6** [comment]
Also in Fixes# you need to give issue number. No need to give PR number in fixes.

**Comment by somya-kapoor** [comment]
@jalajdorai Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this issue.

**Pull request closed without merging**